snapshot/bundlestore/server.go

```diff
@@ -2,144 +2,66 @@ package bundlestore
 
 import (
 	"fmt"
-	"io"
 	"net/http"
 	"regexp"
-	"strings"
-	"time"
 
 	log "github.com/sirupsen/logrus"
 
+	"github.com/twitter/scoot/bazel"
+	"github.com/twitter/scoot/bazel/cas"
 	"github.com/twitter/scoot/common/stats"
+	"github.com/twitter/scoot/snapshot/store"
 )
 
+// Represents a Bundlestore Server that serves HTTP and gRPC Bazel CAS APIs
```

Comment:
The naming change from bundlestore -> store seems incomplete? When are we choosing to use one or the other?

Reply:
I'm not renaming bundlestore to store, I just separated Store into its own package so it is portable. If anything bundlestore should be apiserver or blobserver or storeserver or something now.